webapp/channels/src/components/widgets/icons/fa_warning_icon.tsx

```diff
@@ -8,21 +8,17 @@ import LocalizedIcon from 'components/localized_icon';
 import {t} from 'utils/i18n';
 
 type Props = {
-    additionalClassName: string | null;
+    additionalClassName?: string;
 }
 
-export default class WarningIcon extends React.PureComponent<Props> {
-    public static defaultProps: Props = {
-        additionalClassName: null,
-    };
-
-    public render(): JSX.Element {
-        const className = 'fa fa-warning' + (this.props.additionalClassName ? ' ' + this.props.additionalClassName : '');
-        return (
-            <LocalizedIcon
-                className={className}
-                title={{id: t('generic_icons.warning'), defaultMessage: 'Warning Icon'}}
-            />
-        );
-    }
+function WarningIcon(props: Props) {
```

Comment:
Do you mind making this a arrow function? We want to keep the code as consistent as possible, and we prefer to use arrow functions.

Reply:
✅ Fixed it. I also prefer the arrow function, but looking at other icons which already in function component form like `webapp/channels/src/components/widgets/icons/back_icon.tsx` it use regular functions makes me think twice.